buddyboss/buddyboss-platform
# Optimize group member count query

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2327 .

### How to test the changes in this Pull Request:
Navigate to the individual group and it will show Fatal error: Allowed memory size.

1. Go to any group
2. See error, 'allowed memory size'.

### Proof Screenshots or Video

https://prntscr.com/11aka6t

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
